Src/CSharpier/SyntaxPrinter/SyntaxNodePrinters/CheckedExpression.cs

```diff
@@ -10,9 +9,12 @@ public static Doc Print(CheckedExpressionSyntax node)
         {
             return Doc.Concat(
                 Token.Print(node.Keyword),
-                Token.Print(node.OpenParenToken),
-                Node.Print(node.Expression),
-                Token.Print(node.CloseParenToken)
+                Doc.Group(
+                    Token.Print(node.OpenParenToken),
```

Comment:
Should we maybe have a helper method for invocation expression like things?

Reply:
I had a similar thought, but it was different enough from how parameters are printed for Invocations that I didn't think it made sense to try to combine them. I just looked around and there are a lot of SyntaxNodePrinters that print the same style for the parens, but they all seem different enough that it would be hard to share the logic. : /